static/js/place/main.tsx

```diff
@@ -93,16 +103,52 @@ class MainPane extends React.Component<MainPanePropType> {
           if (isOverview && Object.keys(this.props.pageChart).length > 1) {
             subtopicHeader = (
               <h3 id={topic}>
-                <a href={`/place/${this.props.dcid}?topic=${topic}`}>{topic}</a>
+                <a href={`/place/${this.props.dcid}?topic=${topic}`}>
+                  <IntlProvider
```

Comment:
Is it possible to wrap this at a higher level - say the whole page component?  I think we should be able to declare this only once in the page.

Reply:
yep, I think I did it. LMK if you wanted it even higher